<launch>
  <include file="$(find robocup_control)/launch/player_control.launch"/>
  <node name="test_thread" pkg="robocup_control" type="dummy_thread" respawn="false" />
  <node name="player_thread" pkg="robocup_control" type="player_thread" respawn="false"/>
</launch>
